common/test/acceptance/pages/lms/fields.py

```diff
@@ -249,3 +249,10 @@ def error_for_field(self, field_id):
         """
         query = self.q(css=u'.u-field-{}.error'.format(field_id))
         return True if query.present else False
+
+    def get_social_first_element(self):
```

Comment:
In python2 and python3 rendering of twitter, facebook and linkedin is different. 
Test is written in hardcoded way thats why failing in py3.

So I picked the first element and pass that value for test.